superset/db_engine_specs/exasol.py

```diff
@@ -0,0 +1,27 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=C,R,W
+
+from superset.db_engine_specs.base import BaseEngineSpec
+
+
+class ExasolEngineSpec(BaseEngineSpec):
+    """Engine spec for Exasol"""
+
+    engine = "exa"
+    max_column_name_length = 128
+    odbc_row_results = True
```

Comment:
When adding new properties, you should add a default value (`False`) to `BaseEngineSpec` in `base.py`.

Reply:
At a minimum, all engine specs should define at least a minimum set of time grains. Take a look at e.g. `sqlite.py`.

Reply:
Thanks, I'll check and update accordingly.

Reply:
Btw, If memory serves me right Exasol is inherently an Oracle dialect, so you might be best off extending the Oracle spec.

Reply:
Afaik this is not the case, Exasol borrows heavily from postgres (e.g. `DATE_TRUNC` which I used to implement the time grain functions is the same in both), but is not a clear descendant in the way that for example Redshift is.